Dockerfile

```diff
@@ -3,12 +3,17 @@ RUN pip install poetry
 COPY poetry.lock pyproject.toml ./
 
 FROM base as production
+RUN poetry config virtualenvs.create false --local && poetry install
 RUN poetry add gunicorn
 RUN poetry install
 COPY ./todo_app ./todo_app
+COPY ./entrypoint.sh /entrypoint.sh
 ENV WEBAPP_PORT=80
 EXPOSE ${WEBAPP_PORT}
-ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:80", "todo_app.app:create_app()"]
+ENV PORT=80
```

Comment:
Nice, setting a default so that your Readme instructions still work out the box is a good idea